src/SwedbankPay.Sdk/PaymentInstruments/Invoice/InvoicePaymentCancelRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace SwedbankPay.Sdk.PaymentInstruments.Invoice
+{
+    public class InvoicePaymentCancelRequest
+    {
+        public InvoicePaymentCancelRequest(ICancelTransaction transaction)
+        {
+            Transaction = Transaction;

```

Comment:
Codacy found an issue: [Remove or correct this useless self-assignment.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)